src/commands/emulators-start.ts

```diff
@@ -17,7 +28,58 @@ module.exports = new Command("emulators:start")
       throw e;
     }
 
-    utils.logLabeledSuccess("emulators", "All emulators started, it is now safe to connect.");
+    const guiInfo = EmulatorRegistry.getInfo(Emulators.GUI);
+    const guiUrl = `http://${guiInfo?.host}:${guiInfo?.port}`;
+    const head = ["Emulator", "Host:Port", "Log File"];
+
+    if (guiInfo) {
+      head.push("View in UI");
+    }
+
+    const uiTable = new Table();
+    uiTable.push([`${clc.green('✔')}  All emulators ready! View status and logs at ${stylizeLink(guiUrl)}`]);
```

Comment:
Sorry but again, please do not print GUI urls unless GUI is running. Alternative text can be `All emulators ready! It is now safe to connect your apps.`

Also nit: rename this to `successMessageTable` to be more general, i.e. not attached to UI.

Reply:
whoops thanks